spec/features/admin/invoices/show_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Invoices Show Page' do
+    describe 'User Story 33' do
+        before do
+            @nico = Customer.create!(first_name: "Nico", last_name: "Shantii")
+            @wolf = Customer.create!(first_name: "Wolf", last_name: "Goode")
+            @invoice_1 = Invoice.create!(status: "Not Paid", customer_id: @nico.id)
+            @invoice_2 = Invoice.create!(status: "Paid", customer_id: @nico.id)
+        end
+
```

Comment:
be careful here because the invoice statuses are supposed to read as cancelled, completed, or in-progress.  Hoa also created enums for the invoice status in the invoice model (see screenshot). So basically if you put one of those numbers in place of "paid" or "not paid" it should read as one of the expected statuses mentioned above.
![Screenshot 2024-02-22 at 5 50 28 PM](https://github.com/odellmac4/little-shop-7/assets/149460430/5ea79e32-53e4-4c67-b5c3-0f4975cbda22)

Reply:
ok got it. I did not know what the enum references would be. Thanks for sharing that. Makes sense to do it that way.

Reply:
I would prefer use string instead of integer. I only use the integer for query.